packages/actions-test/src/event.ts

```diff
@@ -53,6 +55,8 @@ export class TestTransactionEvent implements TransactionEvent {
         this.input = "0x"
         this.value = "0x"
         this.nonce = "0x"
+        this.transactionHash = "0x"
+        this.alertId = "-"
```

Comment:
Maybe generate a fake uuid? I'm not sure what the zero value would be here.

Reply:
Honestly, I don't think anything depends on alertId here except our FE attempting to parse it and link so we can remove this from API, and just keep it as parameter in payload (keep it in Conjure TransactionPayload, but not here in TransactionEvent)